obal/data/roles/setup_sources/tasks/git.yml

```diff
@@ -10,8 +10,3 @@
     version: "{{ git_branch | default(omit) }}"
     force: true
     depth: 1
-
-- name: 'Register source dir tito argument'
-  set_fact:
-    build_package_tito_releaser_args:
-      - 'source_dir={{ setup_sources_git_checkout_dir }}'
```

Comment:
Now that this is dropped, `setup_sources_git_checkout_dir` isn't used anywhere anymore. That means the role now just clones a git repository and then never uses it anymore. Is this role now redundant too? `setup_sources_git` was introduced in 923d72cd0c65ff652d052cb531c06909a286a0c7 but I think this is Pulp  2. https://github.com/pulp/pulp-packaging/blob/master/package_manifest.yaml is the only place I could find that used it.